node/src/main.rs

```diff
@@ -467,6 +467,11 @@ fn async_main() -> impl Future<Item = (), Error = ()> + Send + 'static {
         "conn_pool_size" => store_conn_pool_size,
     );
 
+    let postgres_conn_pool = DieselStore::create_conn_pool(
```

Comment:
I'm not a fan of shortening words unnecessarily myself, can we use `create_connection_pool` for the method?